x-pack/plugin/esql/compute/src/test/java/org/elasticsearch/compute/aggregation/blockhash/BlockHashTests.java

```diff
@@ -1202,7 +1296,9 @@ public void add(int positionOffset, IntVector groupIds) {
                 add(positionOffset, groupIds.asBlock());
             }
         });
-        if (blockHash instanceof LongLongBlockHash == false && blockHash instanceof BytesRefLongBlockHash == false) {
+        if (blockHash instanceof LongLongBlockHash == false
+            && blockHash instanceof BytesRefLongBlockHash == false
+            && blockHash instanceof BytesRef3BlockHash == false) {
```

Comment:
I think this shouldn't be in the `instanceof` because it does do the right thing with nulls.

Reply:
This is to skip the assert lookup.